ingestion-beam/src/main/java/com/mozilla/telemetry/io/Read.java

```diff
@@ -46,8 +48,15 @@ public PubsubInput(ValueProvider<String> subscription) {
     @Override
     public Result<PCollection<PubsubMessage>> expand(PBegin input) {
       return input //
-          .apply(PubsubIO.readMessagesWithAttributes().fromSubscription(subscription))
-          .apply(ToPubsubMessageFrom.identity());
+          .apply(PubsubIO.readMessagesWithAttributesAndMessageId().fromSubscription(subscription))
+          .apply(MapElements.via(new SimpleFunction<PubsubMessage, PubsubMessage>() {
+
+            public PubsubMessage apply(PubsubMessage input) {
```

Comment:
`input` is the variable name for the input to the outer `expand` method, so it's best to avoid masking that here.